indico/modules/events/registration/templates/display/conference_home.html

```diff
@@ -33,9 +34,20 @@
                     {%- trans %}Registration{% endtrans %}
                 {%- endif -%}
             </div>
-            {% if user_has_registered %}
-                <div>{% trans %}You have registered for this event.{% endtrans %}</div>
+            {% if user_registrations %}
+                {% if user_registrations < regforms|length %}
+                    <div>{% trans %} You have registered for this event, but there are more registration forms available.{% endtrans %}</div>
+                {% else %}
+                    <div>{% trans %} You have registered for this event.{% endtrans %}</div>
+                {%  endif %}
                 <div class="toolbar">
+                    {% if is_number_published %}
+                        <span class="i-label icon-user" title="{% trans %}Total registrations{% endtrans %}">{{ regforms[0].active_registrations|length }}
```

Comment:
`regforms[0]` is used quite often, I'd go for something like this:

    {% set regform = regforms|first if regform|length == 1 else none %}

or

    {% set regform = regforms[0] if regform|length == 1 else none %}

Personally I prefer `|first` since it also works e.g. for a single-element set and not just for a list.

Or `single_regform` instead of `regform` if you want to be more verbose.